msteams-platform/m365-apps/prerequisites.md

```diff
@@ -25,15 +25,25 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
 
 You'll also need to enable sideloading for your tenant:
 
-1. Sign in to Microsoft 365 admin center (https://admin.microsoft.com) with your test tenant credentials and select **Teams** from the side panel to open the *Microsoft Teams admin center*.
-1. Select: Teams apps > Manage apps > **Org-wide app settings**.
-1. Under **Custom apps**, toggle on the option *Interaction with custom apps*.
+ 1. Sign in to [Teams admin center](https://admin.teams.microsoft.com/dashboard) with your test tenant credentials.
 
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="Enable sideloading for custom apps from the Teams admin center":::
+ 1. From the left pane, select **Show all**.
+
+ 1. Under admin centers, select **Teams**. A new tab opens the Microsoft Teams admin center.
```

Comment:
If you're linking to admin.teams.microsoft.com, is this still needed? I thought that went directly tot the teams admin center.

Reply:
Yes the link is directed to Teams admin center.

Reply:
Remove it if it's not actually needed, given that the link is now admin.teams.microsoft.com

Reply:
removed it as suggested.